pkg/harvester/l10n/zh-hans.yaml

```diff
@@ -838,8 +982,49 @@ typeLabel:
       one { VM 快照 }
       other { VM 快照 }
     }
+  network.harvesterhci.io.vlanconfig: |-
+    {count, plural,
+      one { 网络配置 }
+      other { 网络配置 }
+    }
   harvesterhci.io.monitoring.alertmanagerconfig: |-
     {count, plural,
-      one { Alertmanager Config }
-      other { Alertmanager Configs }
-    }
\ No newline at end of file
+      one { Alertmanager 配置 }
+      other { Alertmanager 配置 }
+    }
+  harvesterhci.io.logging.clusterflow: |-
+    {count, plural,
+      one { Cluster Flow }
+      other { Cluster Flow }
+    }
+  harvesterhci.io.logging.clusteroutput: |-
+    {count, plural,
+      one { Cluster Output }
+      other { Cluster Output }
+    }
+  harvesterhci.io.logging.flow: |-
+    {count, plural,
+      one { Flow }
+      other { Flow }
+    }
+  harvesterhci.io.logging.output: |-
+    {count, plural,
+      one { Output }
+      other { Output }
+    }
```

Comment:
Maybe it needs to be translated here too, but I don't know what it translates to。。。。    @guangbochen

Reply:
prefer to keep the English version here.